examples/coap-server.c

```diff
@@ -933,6 +1431,175 @@ hnd_unknown_put(coap_context_t *ctx,
   hnd_put(ctx, r, session, request, token, query, response);
 }
 
+#if SERVER_CAN_PROXY
+
+static void
+remove_proxy_association(coap_session_t *session, int send_failure) {
+
+  size_t i;
+
+  for (i = 0; i < proxy_list_count; i++) {
+    if (proxy_list[i].incoming == session) {
+      coap_session_release(proxy_list[i].ongoing);
+      break;
+    }
+    if (proxy_list[i].ongoing == session && send_failure) {
+      coap_session_release(proxy_list[i].ongoing);
+      if (send_failure) {
```

Comment:
`send_failure` will always be true at this point, due to the condition two lines before.

Reply:
Of course - now removed.

Reply:
Code changes pushed